graylog2-web-interface/src/components/common/Delayed.test.tsx

```diff
@@ -46,6 +46,7 @@ describe('Delayed', () => {
 
     expect(queryByText('Hello World!')).toBeNull();
 
+    // @ts-ignore
```

Comment:
I think it's better to actually fix the error. This will do the trick as it will prevent immediate returns:

```
 act(() => { jest.advanceTimersByTime(200); });
```